### DISCUSSION TITLE
Welcome to ThemeSaver Discussions!

### DISCUSSION THREAD

**Discussion by techcoder20** [techcoder20/ThemeSaver#3] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by GerardPolloRebozado** [comment]
Im on ubuntnu and give windows manager error waht windows maangers are compatible?

**Reply by techcoder20** [reply]
Hi I have recently added initial support for gnome. It doesnt support exporting and importing but everything else works normally.
